schematic/utils/validate_utils.py

```diff
@@ -7,15 +7,15 @@
 def validate_schema(schema):
     """Validate schema against schema.org standard
     """
-    json_schema_path = os.path.join('validation_schemas', 'schema.json')
+    json_schema_path = os.path.join('data', 'validation_schemas', 'schema.json')
     json_schema = load_json(json_schema_path)
     return validate(schema, json_schema)
 
 
 def validate_property_schema(schema):
     """Validate schema against SchemaORG property definition standard
     """
-    json_schema_path = os.path.join('validation_schemas', 'property_json_schema.json')
+    json_schema_path = os.path.join('data', 'validation_schemas', 'property_json_schema.json')
```

Comment:
These file paths still assume that the working directory is the `schematic` repository. Are these files important to `schematic` users or developers? If the former, I would add them to the configuration file. If the latter, then it might be worthwile starting a second developer-focused configuration file.

Reply:
You're right, it would be better to start a developer configuration file and allow these paths to be configured there. I'll work on that on a separate PR @BrunoGrandePhD.